Shape_detection/include/CGAL/Shape_detection/Efficient_RANSAC/Octree.h

```diff
@@ -84,6 +105,13 @@ class Indexed_octree : public Octree::Octree<typename Traits::Input_range, typen
   std::size_t maxLevel() const {
     return this->max_depth_reached();
   }
+
+  void createTree(double cluster_epsilon_for_max_level_recomputation = -1., std::size_t bucketSize = 2,
```

Comment:
no CamelCase.

Reply:
Fixed! That was a carry-over from the original RANSAC octree.

Note: I intend for `Direct_octree` and `Indexed_octree` to be very temporary. They're acting as thin wrappers for the new octree, but as soon as I get the code to pass tests I'm hoping to refactor it so that it uses the octree more directly.